ci/compliance/report.sh

```diff
@@ -2,16 +2,6 @@
 
 set -e
 
-duvet report \
-    --spec-pattern 'specs/**/*.toml' \
-    --spec-pattern 'ci/compliance/specs/**/*.toml' \
-    --source-pattern 'h3/**/*.rs' \
-    --workspace \
-    --exclude duvet \
-    --require-tests false \
-    --blob-link "https://github.com/hyperium/h3/blob/master" \
-    --issue-link 'https://github.com/hyperium/h3/issues' \
-    --no-cargo \
-    --html ci/compliance/report.html
+duvet report
```

Comment:
Do we need the --ci parameter?

Reply:
duvet should now look at the the `CI` environment variable, which is set by [GitHub actions by default](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables). The nice thing about using the env var is the same script can be used locally to update the snapshot instead of checking it.